matrix-meetings-widget/src/components/meetings/MeetingsPanel/MeetingsPanel.tsx

```diff
@@ -57,6 +57,13 @@ export const MeetingsPanel = () => {
       widgetApi.widgetParameters.userId,
     ),
   );
+  const { canSendMessageToAllBreakoutSessions } = useAppSelector((state) =>
+    selectRoomPermissions(
+      state,
+      widgetApi.widgetParameters.roomId ?? '',
+      widgetApi.widgetParameters.userId,
+    ),
+  );
```

Comment:
I don't understand why the selector needs to be called again. Why not get the value from the existing call?

```diff
- const { canCreateBreakoutSessions } = useAppSelector((state) =>
+ const { canCreateBreakoutSessions, canSendMessageToAllBreakoutSessions } = useAppSelector((state) =>
```